Sources/CleevioUI/Views/Inputs/CleevioInputField.swift

```diff
@@ -79,12 +82,37 @@ public struct CleevioInputField<
             self.isExternallyFocused = isFocused
             self.contentPadding = contentPadding
             self.font = font
+
+            self.errorPresentationCondition = { _, error in error != nil }
```

Comment:
Wouldn't simple `Boolean` suffice? E.g. `showErrorOnFocused` - since it may be the only reasonable usage of this closure; `error != nil` is something you decide when you set `.error` on the `View`

Reply:
Potentially it could be an `OptionSet`, such as `hideErrorIfDisabled`, `hideErrorIfFocused` - and would be easier to add new options.

Reply:
What i wanted to achieve is passing local combination of states "isExternallyFocused" and "isFocused" (which i find a bit redundant), while not breaking API that is already in use and let client to decide, whether the error should be shown or not.
But lets discuss and find better solution.